2018q4/dts.md

```diff
@@ -0,0 +1,9 @@
+## DTS Update
+
+Contact: Emmanuel Vadot <manu@FreeBSD.org>
+
+DTS files (Device Tree Sources) were updated to be in pair with Linux 4.20 for
+head and 4.19 for the 12-STABLE branch.
+
+The DTS are now compile for some arm64 boards as the one present in U-Boot are
```

Comment:
Maybe I'm being too pedantic, but I think this should read "the DTS is now compiled for some arm64 boards, as..."